concordium-consensus/src/Concordium/GlobalState/Persistent/Account/StructureV1.hs

```diff
@@ -2081,11 +2150,24 @@ migrateV3ToV3 acc = do
 migrateV3ToV4 ::
     ( MonadBlobStore m,
       MonadBlobStore (t m),
-      MonadTrans t
+      AccountMigration 'AccountV4 (t m),
+      MonadTrans t,
+      MonadLogger (t m)
     ) =>
     PersistentAccount 'AccountV3 ->
     t m (PersistentAccount 'AccountV4)
-migrateV3ToV4 _acc = error "TODO(drsk) github #1221. implement migrateV3ToV4"
+migrateV3ToV4 acc = do
+    (accountEnduringData, newStakedAmount) <-
+        runStakedBalanceStateTT
```

Comment:
I don't think it's necessary to run this inside `StakedBalanceStateTT`.

Reply:
we removed it now.